openmc/surface.py

```diff
@@ -289,6 +323,37 @@ def x0(self, x0):
         check_type('x0 coefficient', x0, Real)
         self._coeffs['x0'] = x0
 
+    def bounding_box(self, side):
+        """Determine an axis-aligned bounding box.
+
+        An axis-aligned bounding box for surface half-spaces is represented by
```

Comment:
Can these docstrings be differentiated for each surface to describe which dimensions in particular will have a +/- infinite extent in the bounding box?